application.py

```diff
@@ -168,6 +170,25 @@ def del_rst():
     response.headers['Content-Type'] = 'text/plain'
     return response
 
+@app.route('/srv/delete/', methods=['POST'])
+def delete():
+    project = request.form.get('project')
+    filename = request.form.get('filename')
+    project_path = os.path.join(FILES_DIR, project)
+    file_path = os.path.join(FILES_DIR, project, filename + '.rst')
+    
+    try:
+        if os.path.isfile(file_path):
+            os.remove(file_path)
+            del_rst()
+        elif os.path.isdir(project_path):
+            shutil.rmtree(project_path)
```

Comment:
Isn't this line a security vulnerability?

    wget  http://server.com.srv/delete/?project=/var/www